sign/rpc-methods.md

```diff
@@ -18,7 +18,74 @@ error: {
 }
 ```
 
-## Pairings:
+## Pairing:
+
+### wc_pairingDelete
+
+Used to inform the peer to close and delete a pairing. All associated sessions of the given pairing must also be deleted.
+
+**Request**
+
+```jsonc
+// wc_pairingDelete params
+{
+  "code": Int64,
+  "message": string
+}
+
+| Iridium |          |
+| ------- | -------- |
+| TTL     | 86400    |
+| Prompt  | false    |
+| Tag     | 1000     |
+```
+
+**Response**
+
+```jsonc
+// Success result
+true
+
+| Iridium |          |
+| ------- | -------- |
+| TTL     | 86400    |
+| Prompt  | false    |
+| Tag     | 1001     |
+```
+
+### wc_pairingPing
+
+Used to evaluate if peer is currently online. Timeout at 30 seconds
+
+**Request**
+
+```jsonc
+// wc_pairingPing params
+{
+  // empty
+}
+
+| Iridium |          |
+| ------- | -------- |
+| TTL     | 30       |
+| Prompt  | true     |
```

Comment:
We have prompt `true` here but false on `wc_sessionPing` and for the chat ping as far as I can tell. Is this intentional here?

Reply:
Great catch!